modules/otel-troubleshoot-metrics.adoc

```diff
@@ -19,26 +19,59 @@ The Operator creates a `<cr_name>-collector-monitoring` telemetry service that y
 
 .Procedure
 
-. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource:
+. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource (CR):
 
 +
 [source,yaml]
 ----
+# ...
   config: |
     service:
       telemetry:
         metrics:
           address: ":8888" # <1>
+# ...
 ----
 <1> The address at which the internal collector metrics are exposed. Defaults to `:8888`.
 
-// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
-
 . Retrieve the metrics by running the following command, which uses the port-forwarding Collector pod:
 +
 [source,terminal]
 ----
 $ oc port-forward <collector_pod>
 ----
 
-. Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors` depending on the deployment mode of the OpenTelemetry Collector:
```

Comment:
This is a long sentence and as it currently stands I think it might need something to make the pacing sound more natural. I feel like a comma here would probably help, otherwise feel free to rework it in any way you think would be better:
```suggestion
. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors`, depending on the deployment mode of the OpenTelemetry Collector:
```

Another option would be to split it up into two sentences (assuming it still makes sense and is accurate):
```suggestion
. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics. Depending on the deployment mode of the OpenTelemetry Collector, use either `PodMonitors` or `ServiceMonitors`.
```

Reply:
Thank you for the suggestion. I split the sentence into two and moved the second part to appear after/below the code block.